tf-modules/cumulus-rds-tf/main.tf

```
@@ -1,6 +1,9 @@
 terraform {
   required_providers {
-    aws  = ">= 3.5.0"
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"
```

```suggestion
      version = ">= 3.14.1"
```

`main/master` has `3.14.1

---

https://github.com/nasa/cumulus/pull/2053/commits/8f3fd99ce7d1bdbbccbc77adba636c02731ca057